### PR TITLE
Patch/11 may16

### PR DESCRIPTION
**Pull request by johnmay**
Some misc patches... on rendering of course!

### CODE REVIEW

**Comment by egonw** [comment]
I will look at it this weekend latest...

**Comment by johnmay** [comment]
No rush

**Comment by johnmay** [comment]
Rebased to stop Travis moaning.

**Comment by egonw** [comment]
Yeah, never tell me "no rush"... too much other things will interfere then, and then I won't get to it :/

File: tool/sdg/src/main/java/org/openscience/cdk/layout/LayoutRefiner.java [hunk 1]
```diff
@@ -556,6 +557,13 @@ private double bend(AtomPair pair, IntStack stack, Point2d[] coords) {
             for (IBond bond : pair.bndAt) {
                 if (bond.isInRing()) continue;
 
+                // has this bond already been tested as part of another pair
+                AtomPair first = firstVisit.get(bond);
```
Comment (egonw) [inline tool/sdg/src/main/java/org/openscience/cdk/layout/LayoutRefiner.java]: Possible NPE source?

Comment (johnmay) [inline tool/sdg/src/main/java/org/openscience/cdk/layout/LayoutRefiner.java]: Yep - If public would check the params at top of method.

**Comment by johnmay** [comment]
Thanks.

**Pull request closed without merging**